UI/OptionsWnd.h

```diff
@@ -30,7 +31,40 @@ class OptionsWnd : public CUIWnd {
     virtual GG::Rect CalculatePosition() const;
 
 private:
-    typedef void (OptionsWnd::* VolumeSliderHandler)(int, int, int);
+
+    /**SoundOptionsFeedback enables immediate player feedback when
+       sound options are changed.*/
+    class SoundOptionsFeedback {
+    public:
+        SoundOptionsFeedback() :
+            m_effects_button(0),
+            m_music_button(0)
+        {}
+
+        /** Stores a pointer to  the sound effects check box.*/
```

Comment:
I guessed that you meant the extra space in the comment between the words "to" and "the".

That is fixed.